contrib/forms/hpi/FormHPI.class.php

```diff
@@ -67,39 +65,37 @@ function FormHPI($id = "", $_prefix = "")
             //$this->date = $this->get_date();
         }
     }
+
     function populate()
     {
         parent::populate();
         //$this->temp_methods = parent::_load_enum("temp_locations",false);
     }
 
-    function toString($html = false)
+    public function __toString()
```

Comment:
as an aside (and not worth fixing) a tostring should list out all the variables. Again, though, that is not worth fixing. Agree with below code changes except would remove above var $string and do this below:
``````
return "ID: " . $this->id . "\n";
``````